README.md

```diff
@@ -577,6 +577,7 @@ _CMS are a practical way to setup a website with many features. CMS often come w
 
 ### E-commerce
 
+- [Aimeos](https://aimeos.org/) - Ultra fast, Open Source e-commerce framework for building custom online shops, market places and complex B2B applications with Laravel #gigacommerce. ([Demo](https://demo.aimeos.org/), [Source Code](https://github.com/aimeos)) `LGPL-3.0` `MIT` `PHP`
```

Comment:
- license should be formatted as `LGPL-3.0/MIT` (sorry the syntax check should have detected it, it didn't)
- please remove the #gigacommerce hashtag-like from the description (keep descriptions short/readable/concise, no decorations)